cllc-public-app/ClientApp/src/app/components/sep/sep-application/liquor/tabs/drink-amounts/drink-amounts.component.html

```diff
@@ -0,0 +1,198 @@
+
+            <div class="d-flex">
+                <label>Step 4</label>
+                <section>
+                    <p>
+                        Press the "Clear" button to reset the entered values if you would like to use the tool again.
+                    </p>
+                </section>
+            </div>
+
+            <app-conversion-tool title="Beer, Ciders and Coolers"></app-conversion-tool>
+
+            <app-conversion-tool title="Wine"></app-conversion-tool>
+
+            <app-conversion-tool title="Spirits"></app-conversion-tool>
+        </section>
+    </mat-expansion-panel>
+
+
+    <mat-expansion-panel>
+        <mat-expansion-panel-header>
+            <mat-panel-title>
+                NEED HELP TO DETERMINE YOUR DRINK TYPES?
+            </mat-panel-title>
+        </mat-expansion-panel-header>
+        <section>
+            <app-drink-planner></app-drink-planner>
+        </section>
+    </mat-expansion-panel>
+
+    <mat-expansion-panel>
+        <mat-expansion-panel-header>
+            <mat-panel-title>
+                NEED HELP PLANNING YOUR DRINK SIZES?
+            </mat-panel-title>
+        </mat-expansion-panel-header>
+        <section>
+            <h3>Drink Size Chart</h3>
+
+            <p>
+                These are the drink serving sizes for standard events and tasting events. Please note, tasting events
+                are limited to smaller drink sizes for sampling purposes.
+            </p>
+
+            <table class="drink-size-chart">
+                <tbody>
+                    <tr>
+                        <th colspan="2">Beer, Ciders &amp; Coolers</th>
+                        <th colspan="2">Wine</th>
+                        <th colspan="2">Spirits</th>
+                    </tr>
+                    <tr>
+                        <th>Standard</th>
+                        <th>Tasting</th>
+                        <th>Standard</th>
+                        <th>Tasting</th>
+                        <th>Standard</th>
+                        <th>Tasting</th>
+                    </tr>
+                    <tr>
+                        <td>12 oz</td>
+                        <td>4 oz</td>
+                        <td>5 oz</td>
+                        <td>2 oz</td>
+                        <td>1.5 oz</td>
+                        <td>.5 oz</td>
+                    </tr>
+                    <tr>
+                        <td>340 ml</td>
+                        <td>114 ml</td>
+                        <td>142 ml</td>
+                        <td>60 ml</td>
+                        <td>43 ml</td>
+                        <td>15 ml</td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <p>
+                When you submit your application you will need to provide liquor quantities and prices based on standard
+                drink sizes. For help converting tasting sizes and drink prices to standard drink sizes and prices click
+                here.
```

Comment:
This text reads: "For help converting .... click here" 
This suggests that users could click on a link but I don't see any links here. Could you please check LDB site for the relevant link? thanks!